packages/blade/src/utils/fireNativeEvent/index.ts

```diff
@@ -0,0 +1 @@
+export * from './fireNativeEvent.web';
```

Comment:
This may break rn builds. 

To support both rn & web seamlessly we can structure it like so: 

We can also add another file specific for react-native `fireNativeEvent.native.ts` 

And inside of the rn implementation, you can just throw error that fireNativeEvent is only supported on web.


And then you will just be able to export 
```
export * from './fireNativeEvent';
```